keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -472,12 +534,11 @@ and creating new ones, as well as about cluster-level services (e.g. DNS):
 -->
 - CRI Runtime
   - CRI runtimes that are capable of emitting CRI events must be installed and running.
-    - Impact of its outage on the feature: Kubelet will detect the outage and fall back on the current default relisting period to make sure the pod statuses are updated in time.
+    - Impact of its outage on the feature: Kubelet will detect the outage and fall back on the `Generic PLEG` with the default relisting period to make sure the pod statuses are updated correctly.
     - Impact of its degraded performance or high-error rates on the feature:
-        - Any instability with the CRI runtime events stream that results in an error can be detected by the kubelet. Such an error will result in the kubelet falling back to the current default relisting period to make sure the pod statuses are updated in time.
-        - If the instability is only of the form degraded performance but does not result in an error then the kubelet will not fall back to the current default relisting period and will continue to use the CRI runtime events stream. This will result in the kubelet updating the pod statuses with either the CRI runtime events or the increased relisting period, whichever is less.
-    - Without the stable stream CRI events this feature will suffer, and kubelet will fall back to relisting with the current default relisting period.
-		- Kubelet should emit a metric `kube_pod_missed_events` when it detects pods changing state between relist periods not caught by an event.
+        - Any instability with the CRI runtime events stream that results in an error can be detected by the kubelet. Such an error will result in the kubelet falling back to the `Generic PLEG` with default relisting period to make sure the pod statuses are updated in time.
+        - If the instability is only of the form degraded performance but does not result in an error then the kubelet will not be able to fall back to the `Generic PLEG` with default relisting period and will continue to use the CRI runtime events stream. With the changes proposed in the section [Pod Status update in the Cache](#pod-status-update-in-the-cache) should help in handling this scenario.
```

Comment:
What is “default relisting period” here?  The old value, or the new value?

Reply:
The old value. The [default relisting value](https://github.com/kubernetes/kubernetes/blob/07315d10b3718973e5ebcc61cbf0fba8a6ec58a9/pkg/kubelet/kubelet.go#L165) of the generic pleg.